hack/dev-deploy.sh

```diff
@@ -18,34 +16,14 @@ fi
 
 echo "got kc latest image: $kc_latest_image"
 
-cat << EOF > tmp/build/Dockerfile
+cat << EOF > Dockerfile.dev
 FROM ${kc_latest_image} AS build
 FROM scratch
 COPY --from=build / /
```

Comment:
For some reason, I get 
`
ERROR: failed to solve: kbld:kapp-controller-sha256-37dd1f0e39e3fc238ca0f80dae5f9d0b7f1765035b0348731423560c1459cddb: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
` at this step. The same is happening on develop branch so not sure if it's due to the changes I introduced.

Reply:
Seems to be working after restarting laptop.